webapp/public/locales/en/main.json

```diff
@@ -757,6 +758,7 @@
   "studies.copySuffix": "Copy",
   "studies.filters.strictfolder": "Show only direct folder children",
   "studies.filters.showChildrens": "Show all children",
+  "studies.filters.toggleScope": "Show/Hide Variants",
```

Comment:
Key name doesn't match the actual functionality (it toggles between references/all, not specifically variants)
suggestion:
`"studies.filters.toggleStudyType": "Toggle Reference/All Studies"`